<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAFontSetWidget</class>
 <widget class="QWidget" name="SAFontSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Font Set</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonBold">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icon/icons/bold.png</normaloff>:/icon/icons/bold.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonItalic">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icon/icons/italic.png</normaloff>:/icon/icons/italic.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonFontSet">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icon/icons/fontSet.png</normaloff>:/icon/icons/fontSet.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
